src/main/java/br/usp/pcs/backdoor/CreateMaliciousFile.java

```diff
@@ -0,0 +1,47 @@
+package br.usp.pcs.backdoor;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CreateMaliciousFile {
+    private static final String FULL_PATH = "filename.txt";
```

Comment:
Um nome menos genérico. Pode ser outro, se você quiser.

```suggestion
    private static final String FULL_PATH = "log.txt";
```